components/inputs/input-date.js

```diff
@@ -438,13 +440,16 @@ class InputDate extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(LitEl
 	_handleKeydown(e) {
 		// open dropdown on down arrow or enter and focus on calendar focus date
 		if (e.keyCode === 40 || e.keyCode === 13) {
+			this._openedOnKeydown = true;
```

Comment:
Would be good to default this and `_openCalendarOnly` to `false` in the constructor, otherwise they're `undefined`.